clusterloader2/pkg/framework/framework.go

```diff
@@ -143,22 +143,26 @@ func (f *Framework) ListAutomanagedNamespaces() ([]string, []string, error) {
 	return automanagedNamespacesList, staleNamespaces, nil
 }
 
+func (f *Framework) deleteNS(namespace string, wg *sync.WaitGroup, errList *errors.ErrorList) {
```

Comment:
Please change the signature to

```
func (f *Framework) deleteNamespace(namespace string) error {
```

This is actually a good refactoring as it will get rid of duplicated code.